lib/gettext-setup/merge_pot.rb

```diff
@@ -0,0 +1,9 @@
+# A task to merge pot files located within the locales directory into one
+# pot file for localization. Output is 'completed.pot' in locales directory.
+module GettextSetup
+  module MergePot
+    def self.merge_pot
+      `msgcat locales/*.pot > locales/completed.pot`
```

Comment:
Is completed.pot the final name for all modules ? Will the forge need to be updated ?